scripts/apothecary/apothecary

```diff
@@ -88,7 +88,8 @@ VS_64_BIT_ENV='VC\bin\x86_amd64\vcvarsx86_amd64.bat'
 # paths to android SDK, etc
```

Comment:
Is the idea to remove the formulas from main repo, but keep updating the .a files with new builds? Or is it to remove the .a files entirely from git soon, and depend on external download or manual build?